src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -2016,35 +2082,56 @@ export default class GeFormRenderer extends LightningElement{
             ),
             paymentMethodToken:
                 this.getFieldValueFromFormState(apiNameFor(PAYMENT_AUTHORIZE_TOKEN)),
+            paymentMethodType : PAYMENT_METHODS.CREDIT_CARD,
         });
     }
 
+    accountHolderType() {
+        return this.isDonorTypeContact()
+            ? ACCOUNT_HOLDER_TYPES.INDIVIDUAL
+            : ACCOUNT_HOLDER_TYPES.BUSINESS;
+    }
+
+    selectedPaymentMethod () {
+        return this.getFieldValueFromFormState(PAYMENT_METHOD);
+    }
+
     processPurchaseResponse = async (responseBody) => {
-        const isPurchaseFailed = responseBody.errors;
-        if (isPurchaseFailed) {
+        if (responseBody.errors) {
             this.updateFormStateWithFailedPurchaseCall(responseBody.errors);
             this.handlePurchaseCallValidationErrors(responseBody.errors);
             this.hasFailedPurchaseRequest = true;
         }
 
-        const isPurchaseTimedout = !responseBody.id && responseBody.message && responseBody.status;
-        if (isPurchaseTimedout) {
+        if (this.hasPurchaseTimedOut(responseBody)) {
             this.updateFormStateWithTimedoutPurchaseCall(responseBody);
             this.hasFailedPurchaseRequest = true;
             this.hasPurchaseCallTimedout = true;
             this.formatTimeoutErrorMessage();
         }
 
-        const isPurchaseCreated =
-            responseBody.id && responseBody.status === this.PAYMENT_TRANSACTION_STATUS_ENUM.CAPTURED;
-        if (isPurchaseCreated) {
+        if (this.isPurchaseCreated(responseBody)) {
             this.updateFormStateWithSuccessfulPurchaseCall(responseBody);
             this.hasFailedPurchaseRequest = false;
         }
 
         await this.saveDataImport(this.saveableFormState());
     }
 
+    isPurchaseCreated (responseBody) {
+       return responseBody.id
+           && (responseBody.status ===
+               this.PAYMENT_TRANSACTION_STATUS_ENUM.CAPTURED
+           || responseBody.status ===
+           this.PAYMENT_TRANSACTION_STATUS_ENUM.SUBMITTED
+       );
```

Comment:
Looks like spacing/tabbing is a little off here, 3 spaces instead of 4. Could we also create booleans for these? Quick example of what I'm thinking below.

```
const isCaptured = responseBody.status === this.PAYMENT_TRANSACTION_STATUS_ENUM.CAPTURED;
const isSubmitted = responseBody.status === this.PAYMENT_TRANSACTION_STATUS_ENUM.SUBMITTED;

return responseBody.id && (isCaptured || isSubmitted);
```